deploy/olm-catalog/ocs-operator/manifests/ocs-operator.clusterserviceversion.yaml

```diff
@@ -1911,7 +1911,7 @@ spec:
                   value: noobaa/noobaa-core:5.5.0
                 - name: NOOBAA_DB_IMAGE
                   value: centos/mongodb-36-centos7
-                image: quay.io/ocs-dev/ocs-operator:latest
+                image: quay.io/aruniiird/ocs-operator:aruniiird-ocs-operator-with-Snapshot-changes-v4
```

Comment:
Don't commit these changes, your branch you push to GitHub should be clean of any development changes.

Reply:
Oops! my bad... I was testing it... will revert them...